examples/kebechet/alerting.md

```diff
@@ -0,0 +1,25 @@
+# Alerting
+
+Once we have all metrics collected and stored, we can define the alerts that are going to be triggered. You can get notified of these alerts via different methods:
```

Comment:
change to "You can get notified of these alerts through emails, Slack etc. Our team uses the [GitHub Alertmanager](https://github.com/m-lab/alertmanager-github-receiver) to handle all the alerts. The GitHub Alertmanager is a webhook receiver that creates GitHub issues from alerts. To learn more about setting up the GitHub Alertmanager, you can refer to our [documentation](https://www.operate-first.cloud/operations/sre/incident-management/github-receiver-setup.md)."